### ISSUE TITLE
[Accessibility][Product Carousel][Related Products] Elements have inappropriate "displaycard" role

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1762]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Open the developer tools panel.
3. View the Product Carousel elements or Related Product cards.

**Current Behavior**
The "displaycard" role applied to Carousel Elements and Related Product cards is not a valid ARIA role. No role information is conveyed to NVDA users.

**Expected behavior/code**
The "displaycard" role should be removed from these elements. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Assistive technology users

**WCAG SC**
4.1.2 Name, Role, Value

**Code snippet**
```
<div role="displaycard" id="productcarousel-e8f61e7594-item-d8f7aa6253" data-cmp-data-layer="{&quot;productcarousel-e8f61e7594-item-d8f7aa6253&quot;:{&quot;xdm:SKU&quot;:&quot;24-MB02&quot;,&quot;xdm:listPrice&quot;:59.0,&quot;xdm:currencyCode&quot;:&quot;USD&quot;,&quot;dc:title&quot;:&quot;Fusion Backpack&quot;,&quot;repo:modifyDate&quot;:&quot;2021-02-11T17:13:13Z&quot;,&quot;@type&quot;:&quot;core/cif/components/commerce/productlistitem&quot;}}" class="card product__card">
    ...
</div>
```
**Possible Solution**
For the Carousel, use the role "group" and follow the carousel authoring practices from W3C. For the Related Products, no role is necessary.

**Additional context**
Carousel Authoring Practices: https://www.w3.org/TR/wai-aria-practices-1.1/examples/carousel/carousel-1.html

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by mhaack** [comment]
Fixed by CIF-2279 via https://github.com/adobe/aem-core-cif-components/pull/666
